docs/platform/tutorials/currency-alerting.md

```diff
@@ -94,112 +57,186 @@ Let’s set up each function one by one, starting with our input function.
+![Pushover Logs](../images/tutorials/currency-alerting/success-pushover.png)
 
+* Don't worry if the logs only show "_Listening to Stream_" initially—remember that the Threshold service only writes a message to the `currency-rate-alerts` topic when the threshold has been crossed.
+* This means that the `currency-rate-alerts` stream might be empty for a short while.
+* Depending on your threshold, it might take a couple of minutes for messages to start coming through (depending on you threshold)
 
-![alt_text](images/image6.png "image_tooltip")
 
+And thats it! Congratulations for making it this far. 
 
-Congratulations-At this point you should be receiving messages to your mobile phone. Congratulations you’ve done it!
+!!! Success "Recap - What did you just do?"
 
+    Here's what you accomplished in this tutorial:
 
-## Recap - What did you just do!
+	✅ You created a real time, always on solution on the Quix serverless compute environment.
 
-* 
-[x] You created a real time, always on solution on the Quix serverless compute environment. All for Free!
+    ✅ You deployed three serverless functions. One to read data from another platform, one to make decisions based on that data, and one to send push notifications to your phone. In real time.
 
-* 
-[x] You deployed two connectors. One to read data from another platform and one to make decisions based on that data and send notifications via SMS. In real time.
+    ✅ You gained some experience in navigating the Quix platform and how to deploy functions—all without writing any code.
 
-* 
-[x] You gained some experience of navigating the Quix portal and how to deploy connectors. All without doing any coding!
 
 ## What’s Next
 
 What else can you use Quix for?
 
 You can stream any kind of data into the Quix platform, from other apps, websites and services. From hardware, machinery or wearables or from anything you can think of that outputs data. You can then process that data in any imaginable way.
 
-See what you can do with it and please share your experiences with us. In fact, share it with your colleagues, friends and family. Blog, tweet and post about it and tell anyone who will listen about the possibilities.
+To learn more, try one of these other tutorials:
+
+* [Build a live video processing pipeline using the Transport for London (TfL) traffic cameras and the YOLO ML model for object detection](ImageProcessing.md)
+* [Perform sentiment analysis on a stream of Tweets about a given subject](sentimentAnalysis.md)
+* [Gather and processes data from an RSS feed and get an alert when specific criteria are met](RSSProcessingPipeline.md)
+* [Stream and visualize real-time telemetry data with an Android app and Streamlit](telemetry-data.md)
+
+
+!!! tip "Getting Help"
```

Comment:
I prefer to mention how to get help at the beginning and the end.

Reply:
Reworded help section and added to top as well.